.travis.yml

```diff
@@ -0,0 +1,29 @@
+sudo: true
+language: node_js
+node_js:
+  - "8"
+
+branches:
+  only:
+    - /.*/
+
+before_install:
+  - curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+  - echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+
+install:
+  - sudo apt update && sudo apt install -y yarn
```

Comment:
install is the default command of the Yarn, could be changed to yarn --production=false